docs/contributing/docs-contributions.md

```diff
@@ -46,40 +46,6 @@ For the purposes of an accessible document outline, content headings should go i
      > Note: If your issue and/or PR doesn't meet the above contribution criteria, it may receive a comment reminding you to do so. If, after two weeks, these updates haven't been made, your issue and/or PR may be closed, which helps us triage issues and PRs efficiently. You can request that it be reopened if and when you are ready to make the updates required.
 3. GitHub then allows you to commit the change and raise a PR right in the UI. This is the _easiest_ way you can contribute to the project!
 
-### Converting a document from a stub
```

Comment:
Deleted this section because it's covered under [How to write a stub](https://www.gatsbyjs.org/contributing/how-to-write-a-stub/#converting-a-stub-to-a-doc), which goes over item 1. Item 2 is no longer needed, and Item 3 is covered later in ["adjusting breadcrumb titles"](https://www.gatsbyjs.org/contributing/docs-contributions/#adjusting-breadcrumb-titles)

Reply:
I'm not sure about deleting all of it. IMO we should keep some kind of reference here as this is still docs contribution, just adjust sections of it as needed